search_gov_crawler/search_gov_spiders/spiders/domain_spider.py

```diff
@@ -1,81 +1,56 @@
-from scrapy.linkextractors import LinkExtractor
-from scrapy.spiders import CrawlSpider, Rule
-from ..items import SearchGovSpidersItem
-from . import domain_spider_helper_variables as helper_variables
-from . import domain_spider_helper_functions as helper_functions
-
-# scrapy command for crawling domain/site
-# scrapy crawl domain_spider -a domain=desired_domain -a urls=desired_url
-# ex: ... -a domain=travel.dod.mil -a urls=https://travel.dod.mil
-
-from scrapy.http import Response
-import re
-from ..items import SearchGovSpidersItem
+from typing import Optional
 
+from scrapy.spiders import CrawlSpider, Rule
 
-def valid_content_type(type_whitelist, content_type_header):
-    content_type_header = str(content_type_header)
-    for type_regex in type_whitelist:
-        if re.search(type_regex, content_type_header):
-            return True
-    return False
+import search_gov_crawler.search_gov_spiders.helpers.domain_spider as helpers
```

Comment:
The absolute imports did not work for me, but using the relative imports (i.e from ..helpers import domain_spider as helpers) did